c++/src/capnp/compat/websocket-rpc.c++

```diff
@@ -0,0 +1,107 @@
+#include <capnp/compat/websocket-rpc.h>
+#include <kj/io.h>
+#include <capnp/serialize.h>
+
+namespace capnp {
+
+WebSocketMessageStream::WebSocketMessageStream(kj::WebSocket& socket, size_t maxMessageReceiveSize)
+  : socket(socket),
+    maxMessageReceiveSize(maxMessageReceiveSize)
+  {};
+
+kj::Promise<kj::Maybe<MessageReaderAndFds>> WebSocketMessageStream::tryReadMessage(
+    kj::ArrayPtr<kj::AutoCloseFd> fdSpace,
+    ReaderOptions options, kj::ArrayPtr<word> scratchSpace) {
+  return socket.receive(maxMessageReceiveSize)
+    .then([options, scratchSpace](auto msg) -> kj::Promise<kj::Maybe<MessageReaderAndFds>> {
+      KJ_SWITCH_ONEOF(msg) {
```

Comment:
We normally intend line continuations by 4 spaces, but we indent the body of a lambda by two spaces past the parent block. So this should be:

```suggestion
      .then([options, scratchSpace](auto msg) -> kj::Promise<kj::Maybe<MessageReaderAndFds>> {
    KJ_SWITCH_ONEOF(msg) {
```

And then rest of the lambda body should be un-indented two spaces. It looks like several other uses of lambdas in this PR should be updated similarly.